app/models/track_sheet.rb

```diff
@@ -95,8 +95,8 @@ def generate_tracking_sheet(druid, pdf)
   def doc_to_table(solr_doc)
     doc = solr_doc.with_indifferent_access # solr doc always has string keys, SolrDocument::FIELDS can be strings or symbols
     table_data = []
-    labels = doc[SolrDocument::FIELD_TITLE]
-    label = labels.blank? ? '' : labels.first.truncate(110)
+    label = doc[SolrDocument::FIELD_TITLE].is_a?(Array) ? doc[SolrDocument::FIELD_TITLE].first : doc[SolrDocument::FIELD_TITLE]
```

Comment:
While in practice I'm seeing this value as a string, the tests indicate that an array may be possible. Update the logic to account for that possibility.

Reply:
That is just bad test setup. It will never be multivalued. See https://github.com/sul-dlss/argo/blob/cd655c7e56afc4d9dce41e3a765039ae001c126c/solr_conf/conf/schema.xml#L22

The `TrackSheet` class should be using `doc.title_display` (https://github.com/sul-dlss/argo/blob/main/app/models/solr_document.rb#L67C14-L67C27) and not `SolrDocument::TitleField`

Reply:
It looks like the `title_display` attribute was never changed from `Array` to `String` when the `SolrDocument::TitleField` was changed.  That should be done too.

Reply:
This is helpful @jcoyne thank you. Updating.

Reply:
@aaron-collier I think it also makes sense to remove `SolrDocument#title` as that is now just the same as `title_display`